test/Interpreter/FunctionConversion.swift

```diff
@@ -84,19 +84,19 @@ func a1(s: AddrOnly?) -> AddrOnly {
 
 FunctionConversionTestSuite.test("Optional") {
   let g11: (Trivial) -> Trivial? = t1
-  let g12: (Trivial!) -> Trivial? = t1
+  let g12: (Trivial?) -> Trivial? = t1
 
   expectEqual(22, g11(Trivial(n: 11))?.n)
   expectEqual(24, g12(Trivial(n: 12))?.n)
 
   let g21: (Loadable?) -> Loadable? = l1
-  let g22: (Loadable!) -> Loadable? = l1
+  let g22: (Loadable?) -> Loadable? = l1
```

Comment:
This line is redundant now

Reply:
Hmm, I thought I caught these and removed them. Thanks for pointing them out.